### ISSUE TITLE
Adapt IconPie's UX and feature set to suit the needs of LibreOffice / KDE icon designers

### ISSUE THREAD

**Issue by ngirard** [GarkGarcia/icon-pie#14]
This is a follow-up of #7.

@GarkGarcia, I know from [this bug report](https://bugs.documentfoundation.org/show_bug.cgi?id=124940) and [this blog post](https://colibreoffice.wordpress.com/2019/04/25/bugfixing-done/) that Andreas Kainz, who's designing icons for both the KDE project and LibreOffice, would appreciate some help and possibly some new tooling to improve his workflow.

So I thought I'd ask Andreas (@DarkknightAK) to share his thoughts & needs.

@DarkknightAK, I'm pretty sure IconPie has a great potential to suit your needs, as you may figure out by browsing the README file and the issues.

@DarkknightAK, could you please tell us if this excerpt from the abovementioned blog post correctly describes your needs ?

> All main icon themes are available in svg, but the svg2png libreoffice renderer give bad quality, so we have to ship the png icons. If we could use the svg icons instead of the png, it mean less work for the designers and the posibility to have color scheme support for icons. 100% dark icon theme support and full color support. What a great improvement.

@DarkknightAK, how would you ideally want IconPie to work ? Could you give us an example of a command-line you wish you would type ?

**Comment by GarkGarcia** [comment]
Yep, `favicons` should definitively be a part of this app. I guess we have a similar problem to https://github.com/GarkGarcia/icon-pie/issues/7#issuecomment-526922072: in the current scheme there wouldn't be a way to specify which source to use to each platform.

How would you like this feature to work (from the user's perspective)?

**Comment by GarkGarcia** [comment]
I created `[freek](https://github.com/GarkGarcia/freek)` to address this issue. However, I'd like to have some input from interested parties before I release it's first official version.
